engine/src/main/java/org/terasology/network/internal/ServerImpl.java

```diff
@@ -322,19 +327,32 @@ private void processBiomeChanges(NetData.NetMessage message) {
             }
         }
     }
+    
+    private void processExtraDataChanges(NetData.NetMessage message) {
```

Comment:
As this is mostly analogous to `processBlockChanges` and `processBiomeChanges`, should I add javadoc for those, too? It would seem odd having only one of them commented, but the others are unrelated to this PR.

Reply:
If you don't mind sure - always good to end up with a higher ratio of javadoc than before! ;-)